lms/djangoapps/badges/models.py

```diff
@@ -220,7 +220,7 @@ class CourseEventBadgesConfiguration(ConfigurationModel):
         blank=True, default='',
         help_text=_(
             u"On each line, put the number of completed courses to award a badge for, a comma, and the slug of a "
-            u"badge class you have created with the issuing component 'edx__course'. "
+            u"badge class you have created with the issuing component 'openedx__course'. "
```

Comment:
"with" -> "that has":
"...badge class you have created that has the issuing component..."